src/app/spaces/space.service.ts

```diff
@@ -104,8 +104,8 @@ export class SpaceService {
       .catch((error: any): Observable<Space> => this.handleError(error));
   }
 
-  deleteSpace(space: Space): Observable<Space> {
-    const url: string = `${this.spacesUrl}/${space.id}`;
+  deleteSpace(space: Space, skipCluster: boolean = false): Observable<Space> {
+    const url: string = `${this.spacesUrl}/${space.id}?skipCluster=${skipCluster}`;
```

Comment:
I'd rather use `URLSearchParams` here, rather than manually constructing the query string

Reply:
Sounds good. It should be done now.